src/data/commandSyntaxMap.ts

```diff
@@ -0,0 +1,56 @@
+export type SyntaxPart = {
+  syntax: string;
+  doc: string;
+};
+
+export type CommandSyntax = {
+  parts: SyntaxPart[];
+};
+
+export type SyntaxMap = {
```

Comment:
Can't we utilise existing? It won't be feasible to maintain two separate maps in same repo.
```
export interface DiceCmdMeta {
  title: string;
  syntax: string;
  body: string;
  url?: string;
}
```

Reply:
I was thinking of migrating from this CmdMeta to later format only. But in a separate issue, it gives us more control. 

I don't want to maintain two metaData as well, I'm leaving the previous cmdMeta for now, because I don't want anything to break while we transition to the other format. 

we can extend my format in a way that it will be compatible with search functionality as well

Reply:
@lucifercr07 do we agree on this?